### PR TITLE
Modified configuration and dependencies archives to automate nodemail…

### PR DESCRIPTION
**Pull request by matiassequeira**
Jorge, would you please check in detail configuration files that I've modified? I want to make sure about them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,5 +6,8 @@
 	"scripts": {
 		"docker-up": "docker-compose up",
 		"docker-down": "docker-compose down"
+	},
+	"dependencies": {
```
Comment (jchiavaro) [inline package.json]: You should add this dependencies to the package.json inside the sever folder: "server/package.json"

File: server/Dockerfile [hunk 2]
```diff
@@ -9,6 +9,7 @@ RUN npm install --global nodemon
 # For npm@5 or later, copy package-lock.json as well
 COPY package.json .
 RUN npm install
+RUN npm install nodemailer --save
```
Comment (jchiavaro) [inline server/Dockerfile]: Remove this line. When docker starts building the image, it runs 'npm install' and it installs all the dependencies specified in package.json. You do not need to install it "manually" here. "npm install nodemailer --save" will install and save the dependency into the package.json file. This is useful when you develop in your local env, but it's not needed here.

File: server/helper/mailSender.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    if(receiversList.length > 0){
+        mailOptions.to= receiversList[0];
+    }
+    else {
+        return errorMessage = 'Destination mail not provided';
+
+    }
+
+    //get the rest of destinations if any
+    for (var i=1; i<receiversList.length; i++){
```
Comment (jchiavaro) [inline server/helper/mailSender.js]: Use forEach method for iterating the array:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

File: server/helper/mailSender.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        mailOptions.to= mailOptions.to + ", " + receiversList[i];
+    }
+
+    console.log('Enviando a: %s', mailOptions.to);
```
Comment (jchiavaro) [inline server/helper/mailSender.js]: Change to english here

File: server/helper/mailSender.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    mailOptions.subject=subject;
+    mailOptions.text=text;
+    mailOptions.html=html;
+
+    //this field doesn't need to be the user used for authentication with the service
+    mailOptions.from= user;
+
+    console.log('Sending email from %s to %s', mailOptions.from, mailOptions.to);
+
+    // send mail with defined transport object
```
Comment (jchiavaro) [inline server/helper/mailSender.js]: Remove this comment. Avoid to comment things that are pretty obviuos

File: server/mailConnection.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+const nodemailer = require('nodemailer');
+
+//CREDENTIALS
```
Comment (jchiavaro) [inline server/mailConnection.js]: Remove comments

File: server/mailConnection.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+//user is only for authentication and does not affect the email content (from field)
+const user= 'ee4p6d6bz7p5fp6w@ethereal.email';
+const password= '3CHv9qA6HAFBfrmmEW';
+
+
+//set credentials and server
```
Comment (jchiavaro) [inline server/mailConnection.js]: remove comments

File: package.json (hunk 1 above)
Comment (matiassequeira) [reply package.json]: This dependency was added automatically by node when I installed nodemailer. I'll leave it in the archive you mentioned. Also, node modified package-lock.json, is it ok?

File: server/Dockerfile (hunk 2 above)
Comment (matiassequeira) [reply server/Dockerfile]: Ok, sorry. Done

**Comment by matiassequeira** [comment]
Usage example:
var MailSender = require('../../helper/mailSender');
    
    var errorMessage= MailSender.mailSender(["MATIAS SEQUEIRA ms@gmail.com", "juan@gmail.com", "pedro@gmail.com"], "This is the subject", "This is email content", ""); //last field can be an HTML. for this example is empty

    console.log(errorMessage);

**Pull request merged**